services/venice-server/src/main/java/com/linkedin/venice/listener/ReadQuotaEnforcementHandler.java

```diff
@@ -55,7 +52,6 @@ public class ReadQuotaEnforcementHandler extends SimpleChannelInboundHandler<Rou
   private final String thisNodeId;
   private final AggServerQuotaUsageStats stats;
   private final Clock clock;
-  private final ExpiringSet<String> noBucketStores = new ExpiringSet<>(30, TimeUnit.SECONDS);
```

Comment:
I'm not sure I understand what about this refactor allows us to remove this?  Or... is just useless?

Reply:
In my assessment it's useless. It serves only for logging, and we already have a metric for it. If we care about this, we ought to alert on the metric, and if we don't care about it, then we don't need the log anyway 😅